### ISSUE TITLE
Route configuration - missing error handler option

### ISSUE THREAD

**Issue by tplevko** [KaotoIO/kaoto#1173]
### Describe the Bug

The Route Configuration resource type in Kaoto provides options to configure it with intercept, interceptFrom, interceptSendToEndpoint, onException, onCompletion.

The resource should be configurable also with Error Handler according to the camel documentation.

### Steps to Reproduce the Bug or Issue

1. create new Route Configuration resource
2. on this resource - select "add branch"
3. see the "Error handler" is missing in the displayed options

### Screenshots or Videos

Provided options:
![Screenshot from 2024-06-19 12-14-04](https://github.com/KaotoIO/kaoto/assets/4180208/fefa2dc4-bc75-4375-8975-d00e1b2b0af7)

camel documentation:
![Screenshot from 2024-06-19 12-13-51](https://github.com/KaotoIO/kaoto/assets/4180208/f16d5b1c-932e-423d-a0db-ed860a9223eb)

### Platform

- OS: [e.g. macOS, Windows, Linux]
- Browser: [e.g. Chrome, Safari, Firefox]
- Version: [latest]

**Comment by lhein** [comment]
@lordrip @tplevko we have the error handler in the new menu. Is that the one?

**Comment by lordrip** [comment]
No, in YAML, the routeConfiguration errorHandler is part of the main form, it's not a step. It's not covered by the configuration form yet
![image](https://github.com/user-attachments/assets/762aa3b9-f61a-4483-a7d6-862327b6294e)

**Comment by lordrip** [comment]
Implemented as part of the new form library effort

![Image](https://github.com/user-attachments/assets/6d9c8ea2-69a7-4bb5-812d-55a2d900516c)
